### DISCUSSION TITLE
Does this (VR) work cross platform multiplayer with Minecraft Java on PC (flatscreen)?

### DISCUSSION THREAD

**Discussion by samtowers** [QuestCraftPlusPlus/QuestCraft#580] [Q&A] [answered]
See title.

I'm curious as I would want to play with a friend who is on their PC playing flatscreen. 

Cheers!

**Comment by Zircon10086** [comment]
Questcraft emulates an environmet to run the java minecraft, and the java minecraft seems to be the same with the PC version. 
So maybe you can have a try with your friend, as there's a multiplayer option in the menu.

**Comment by jesse-dot** [comment, marked as answer]
yes, you can play with both flat screen players and VR players. QuestCraft uses the vivecraft mod to play VR. you need the vivecraft plugin installed on a server for other flat screen players to see you move your hands. I hope this helps!
